.github/workflows/main.yml

```diff
@@ -72,10 +72,23 @@ jobs:
         run: |
           npm test
           make travis-test-npm
+
+      - name: Cleanup (NPM)
+        if: ${{ always() && matrix.test_type == 'NPM' }}
+        env:
+          S3_ENCRYPT_KEY: ${{ secrets.S3_ENCRYPT_KEY }}
+          # The need for this old environment variable name will go away soon.
+          # Once the new snovault is in, we should retain these for a little while to make sure the right
+          # variable is being used, and then this can go away. -kmp 9-Mar-2021
+          # I have named the ID slightly differently so we can see if the old names are in use.
+          TRAVIS_JOB_ID: ff-npm-x-test-${{ github.run_number }}-
+          # This will be the new environment variable name.
+          TEST_JOB_ID: ff-npm-test-${{ github.run_number }}-
+        run: |
```

Comment:
Oh, this seems particularly bad. This needs to be `sno-` not `ff-` or else test runs on this PR will generate IDs that clash with `ff`. This can be the cause of phantom problems where a particular collision, if test data exists from (or worst case happens during) another's test. Updating the code doesn't fix the problem unless you also purge the potentially colliding data. The alternative is to add an arbitrary versioning sequence number such as by doing `-0-` rather than just `-` at the end of the line. Anything to make newly made tokens not collide with the others.

I apologize for not explaining this prefixing scheme better. The intent is that Fourfront uses the `ff-` prefixes, cgap uses `cgap-` prefixes, and snovault uses `sno-` prefixes. This is one of few places where it actually matters for snovault to have its own identity and not just blend with the identity of its destined application. The (ad hoc) 'application' testing of the `snovault` repo is a distinct 'application' from the 'application' doing testing of the `fourfront` or `cgap`.